qa/suites/upgrade/reef-x/filestore-remove-check/1-ceph-install/reef.yaml

```diff
@@ -0,0 +1,32 @@
+meta:
+- desc: install ceph/reef latest
+tasks:
+- install:
+    exclude_packages:
+      - ceph-mgr-cephadm
+      - cephadm
+      - libcephfs-dev
+    branch: reef
+- print: "**** done install reef"
```

Comment:
Does `filestore-remove-check` test make sense for `reef-x` suite?  Wouldn't this test be starting with a cluster that doesn't support filestore to begin with?

Reply:
Yup, no need for it!